C/kmsg_log.c

```diff
@@ -5,52 +5,55 @@
 #include <fcntl.h>
 #include <string.h>
 
+#define ALT_LOG "/sdcard/lible.log"
+
 int klog(__attribute__((unused)) int level,__attribute__((unused)) char * a )
 {
+
+ char *buf = malloc(sizeof(char) * 256);
```

Comment:
we can make it 
`char *buf; 
*buf = malloc(sizeof(*buf) * 256);
`
im pushing for this because if we ever want to change the type of buf we would have less work and more reliability at hand